.github/workflows/deploy_to_prod.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will update the code for the production environment
+
+on:
+  pull_request:
```

Comment:
you can remove this no?
no need to run this on each pull request, only when something is merged to release